Paper/main.tex

```diff
@@ -127,15 +127,14 @@ \section{RESNET explanation}
 
 \subsection{Training without transfer learning}
 
-The idea behind training the network from scratch is that the network would be more tailored to the given task since it has no previous information stored in the network. This contrasts with transfer learning since there prior information is present.
+The idea behind training the network from scratch is that the network would be more tailored to the given task since it has no previous information stored in the network. This contrasts with transfer learning since there prior information is present. As a model the ResNet18 architecture was used. This architecture was chosen since it is a good compromise between network size and performance.
 \newline
 
 \noindent
 \textbf{Dataset Details:}
 
 \noindent
-To train the network the DIV2K training and validation dataset was used. The dataset contains 800 train images and 100 test images. Since the images had different dimensions, we first had to crop them to specific in this case 512x512. Afterwards we compressed the images to specified file sizes as described in \ref{sec: filesize}. This leads to 10 classes – one for each compression algorithm – for each file size category. We trained the network by using the images that were compressed to a file size of 25kB.
-\newline
+To train the network the DIV2K training and validation dataset was used. The dataset contains 800 train images and 100 test images. Since the images had different dimensions, we first had to crop them to specific in this case 512x512. Afterwards we compressed the images to specified file sizes as described in \ref{sec: filesize}. With the primary objectice beeing to recignize the different compression algorithms the output of the network should represent the different encoding schemes. This leads to 10 classes in each file size category.
```

Comment:
`we first had to crop them to specific...` reads a little bit starnge is there maybe a word missing?

Reply:
rewrote the sentence